geode-core/src/main/java/org/apache/geode/internal/cache/EntryEventImpl.java

```diff
@@ -165,6 +165,9 @@
    */
   private byte[] deltaBytes = null;
 
+  /** If true, causes deltas to trigger recalculation of bucket size **/
+  private boolean forceRecalculateSize = false;
```

Comment:
I'd like to see this new variable removed and instead, in the two places we use it, call delta.getForceRecalculateSize().
You can easily do this on line 1722 because it had just done (on line 1718) a test that v is an instanceof Delta. So add the next line: Delta delta = (Delta)v;

On line 1865 the delta instance is the "value" local variable. You can see this on line 1841 when this code casts "value" to Delta. So I'd recommend you add a local variable at 1841 like so: Delta delta = (Delta)value;
and then use that on 1865 to decide if we should recalculate.
Having the extra state on EntryEventImpl adds complexity and in the future it could get out of sync with Delta.getForceRecalculateSize(). By just calling this new method I think the code will be better.

Reply:
Sorted out; looks like when I attempted this once I grabbed the wrong delta value at one point. Anyway, it works now without the member variable.